Integrations/python/deephaven/types.py

```diff
@@ -0,0 +1,43 @@
+# 
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent Pending
+# 
+
+"""
+Functions to return types from jpy for convenience
+"""
+
+import jpy
+
+# Java primitives
+
+def Byte():
+    return jpy.get_type("byte")
```

Comment:
not super important, but we can avoid repeated call to the function by creating a global constant to represent a Java type.